api/v1/swagger.yaml

```diff
@@ -1988,27 +1997,632 @@ paths:
+        response. If the Refocus configuration parameter `useAccessToken` is
+        set to `true`, you must include an `Authorization` request header with
+        your [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the
+        value. You can get a token using `POST /v1/register` or `POST /v1/tokens`.
+      externalDocs:
+        description: >-
+          GET /generatorTemplates
+        url: http://focus.salesforce.com/docs/api/generatorTemplates
+      operationId: findGeneratorTemplates
+      parameters:
+        -
+          $ref: "#/parameters/GeneratorTemplateFieldsParam"
+        -
+          name: sort
+          in: query
+          description: >-
+            Specify the sort order using a field name, e.g. '...?sort=name'.
+            Prepend the field name with a minus sign to specify descending order,
+            e.g. '...?sort=-name'.
+          required: false
+          type: array
+          collectionFormat: csv
+          items:
+            type: string
+            enum:
+              - description
+              - name
+              - version
+              - createdAt
+              - updatedAt
+              - -description
+              - -name
+              - -version
+              - -createdAt
+              - -updatedAt
+        -
+          $ref: "#/parameters/limitParam"
+        -
+          $ref: "#/parameters/offsetParam"
+        -
+          name: description
+          in: query
+          type: string
+          maxLength: 4096
+          description: >-
+            Filter by the generatorTemplate description; asterisk (*) wildcards ok.
+        -
+          name: name
+          in: query
+          type: string
+          description: >-
+            Filter by the generatorTemplate name; asterisk (*) wildcards ok.
+        -
+          name: version
+          in: query
+          type: string
+          description: >-
+            Filter by the generatorTemplate version; asterisk (*) wildcards ok.
+        -
+          name: tags
+          in: query
+          items:
+            type: string
+            maxLength: 60
+            pattern: ^[0-9A-Za-z_][0-9A-Za-z_\\-]{1,59}$
+          description: >-
+             Comma-separated list of tags to include/exclude. Tag names are
+             case-insensitive. For example, ?tags=FOO,BAR will only return
+             aspects with tags FOO and BAR. Prefix each of the tag name with
+             a negative sign to indicate that a aspect with that tag should
+             be excluded. For example, ?tags=-BAZ,FOO will return only the
+             aspects with tag name not equal to BAZ or FOO. A mix of tag
+             names with and without minus signs, for example ?tags=BAZ,-FOO
+             are not allowed. GeneratorTemplates without tags are not included in the
+             include result set.
+          type: string
+          required: false
+        -
+          name: bulk
```

Comment:
oops "bulk" needs to be moved _inside_ of the connection object as a connection attribute in the db model, and no need to have it here for query parameter filtering